Alignment/OfflineValidation/src/PreparePVTrends.cc

```diff
@@ -663,6 +663,11 @@ pv::biases PreparePVTrends::getBiases(TH1F *hist)
 /*--------------------------------------------------------------------*/
 {
   int nbins = hist->GetNbinsX();
+  // to avoid memory leak flagged by S.A. return default constructed object
```

Comment:
While this is a welcome sanity check, I think that to avoid the memory leak you should delete the `y` and `err` before exiting from the method, instead